packages/app/src/cli/commands/app/function/replay.ts

```diff
@@ -47,19 +45,14 @@ export default class FunctionReplay extends Command {
     watch: Flags.boolean({
       char: 'w',
       hidden: false,
+      allowNo: true,
```

Comment:
Minor tweak, this enables `--no-watch` so that you can turn off watching without using the `env` variable